sharedcode/localFooter.html

```diff
@@ -1,44 +1,42 @@
-<!-- Unit with one address, social, releated links (short) -->
-<div class="wdn-grid-set wdn-footer-links-local">
-	<div class="wdn-col-full bp640-wdn-col-two-thirds bp960-wdn-col-one-half">
-		<div id="wdn_footer_contact" class="wdn-footer-module">
-			<span role="heading" aria-level="1">Lincoln EITC Coalition<span class="wdn-text-hidden"> Contact Information</span></span>
-			<div class="wdn-grid-set-halves bp960-wdn-grid-set-halves">
-				<div class="wdn-col">
-					<dl class="wdn-contact-info">
-						<dt>
-							<span class="wdn-icon-location" aria-hidden="true"></span><span class="wdn-text-hidden">Address</span>
-						</dt>
-						<dd>
-							<strong>University of Nebraska&ndash;Lincoln</strong><br />
-							1400 R Street<br />
-							Lincoln, NE 68588<br />
-						</dd>
-						<dt>
-							<span class="wdn-icon-phone" aria-hidden="true"></span><span class="wdn-text-hidden">Phone Number</span>
-						</dt>
-						<dd>
-							<a href="tel:4024727211">402-472-7211</a>
-						</dd>
-						<dt>
-							<span class="wdn-icon-mail" aria-hidden="true"></span><span class="wdn-text-hidden">Email address</span>
-						</dt>
-						<dd>
-							<a href="mailto:vita@unl.edu">vita@unl.edu</a>
-						</dd>
-					</dl>
-				</div>
-			</div>
-		</div>
-	</div>
-	<div class="wdn-col-full bp640-wdn-col-one-third bp960-wdn-col-one-half">
-		<div id="wdn_footer_related" class="wdn-footer-module">
-			<span role="heading" aria-level="1">Related Links</span>
-			<ul class="wdn-related-links-v1">
-				<li><a href="https://events.unl.edu/">UNL Events Calendar</a></li>
-				<li><a href="https://ucomm.unl.edu/">University Communications</a></li>
-				<li><a href="https://www.unl.edu/ucomm/chancllr/">Office of the Chancellor</a></li>
-            </ul>
-		</div>
-	</div>
-</div>
+<nav id="dcf-footer-group-1" role="navigation" aria-labelledby="dcf-footer-group-1-heading">
```

Comment:
Also, are we able to remove the code in the `sharedcode/affiliate` folder?

Reply:
Yes.